torch_xla/distributed/spmd/xla_sharding.py

```diff
@@ -738,6 +739,7 @@ class XLAPatchedLinear(torch.autograd.Function):
   """
 
   @staticmethod
+  @custom_fwd(device_type='xla', cast_inputs=torch.get_autocast_dtype('xla'))
```

Comment:
Good to add some comments as to why the annotations are added.

Reply:
+1. Can we add this comment to to class description? Also, can you also point out what is torch.get_autocast_dtype?

Reply:
Added in latest commit. Thanks for catching!